### DISCUSSION TITLE
terrible playback of videos backed up on server

### DISCUSSION THREAD

**Discussion by MaBeniu** [immich-app/immich#7330] [Q&A]
Hi, 

I have immich installed on proxmox using docker compose in a privileged container (4gb RAM, 2GB swap, 4cores of N100 Alderlake).
current version of the server and app is the latest 1.95.1. videos are captures with H264 format and no transcoding was performed due to I dont have a need to change original format.
My intention is to store 4k videos on local storage and also be able to play them on the mobile or other device that is on a local network. Local network has wifi 6 mesh setup and internet speeds are not an issues.

What I experience is that a video keeps spinning loading circle every 5-10 seconds. In example trying to open 40 seconds video that is approx 300MB.
I see initial data download speed from the server at moment of opening of video is 42 MB/s which should be sufficient to download it for viewing however after first 10seconds downloading stops (judging by connection speed drop) then after another maybe 10 seconds download starts again.

I checked CPU load of the server and it never reaches more than 25%, immich_server process at some moments getting up to 19% max.

Need an ideas how to troubleshoot where is the issue?

**Comment by mglaske** [comment]
I seem to have the same exact issue with higher resolution videos (4k).

**Comment by MaBeniu** [comment]
with new server and app version 1.100 it started working fine not sure what has changed but on mobile app or pc browser it just works now flawlessly

**Comment by lao9s** [comment]
I’m experiencing the same issue. Playing videos in the browser is a terrible experience — playback stops every 5–10 seconds. On mobile, it works fine.

**Comment by jingteng25742** [comment]
Just setup Immich.app (latest) and it's been fantastic for organizing photos. 
I am also having the video playback issue. 
Video playback is laggy but server cpu usually is below 5%
It just seems like video buffering speed is really slow for some reason.
This is all on a LAN setup.
